src/components/Select/Select.style.ts

```diff
@@ -9,7 +9,6 @@ export const selectWrapper = ({
 }) => (): SerializedStyles => css`
   position: relative;
   min-width: ${rem(150)};
-  max-width: ${rem(620)};
```

Comment:
@kostasdano this will affect the large selects of CMO you have to do it on the product now